rest_framework/authtoken/admin.py

```diff
@@ -4,6 +4,7 @@
 
 
 class TokenAdmin(admin.ModelAdmin):
+    search_fields = ("key", "user__username", "user__email")
```

Comment:
`email` isn't a required field on the `User` model, and DRF never expects it to be defined (unlike `username`). Any particular reason why it's included in here?